.storybook/main.js

```diff
@@ -0,0 +1,13 @@
+module.exports = {
```

Comment:
This make storybook use the preset AND tells webpack to include the correct folder. I guess `src` is the default.

```js
const path = require('path');
const WORKSPACE_ROOT = '../packages';

module.exports = {
  stories: [`${WORKSPACE_ROOT}/**/stories.(ts|tsx)`],
  addons: [
    '@storybook/react',
    '@storybook/addon-actions',
    '@storybook/addon-links',
    {
      name: '@storybook/preset-typescript',
      options: {
        include: [path.resolve(__dirname, WORKSPACE_ROOT)],
      },
    },
  ],
};
```